maintenance/aws-janitor/BUILD

```diff
@@ -37,3 +37,10 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+go_binary(
+    name = "aws-janitor",
```

Comment:
Did you manually create this? Why is there an aws_janitor go_binary() target?

Reply:
No, I ran hack/update-bazel.sh, not sure why it kept the old one. Fixing...

Reply:
probably cause the binary name changed, and it didn't recognize the old one as being autogenerated.

Reply:
done.